packages/app/components/ConnectButton.tsx

```diff
@@ -17,13 +18,24 @@ const CustomConnectButton = ({
   ensName?: string;
   size: SizeProps;
 }) => {
+  const { chain } = useNetwork();
   const { data: avatar } = useEnsAvatar({
     name: ensName,
     chainId: 1,
   });
+
+  const TOKEN_BY_CHAIN: { [chainId: number]: string } = {
+    [ChainId.ETHEREUM]: WETH[1].address,
```

Comment:
**MINOR**, non-blocking:

WDYT to abstract that `1` into a const to get rid of the magic number?

Reply:
yeah, true, we can use `ChainId.ETHEREUM`.